lib/loader/serialization/serialize.cpp

```diff
@@ -0,0 +1,255 @@
+#include "ast/module.h"
+#include "loader/loader.h"
+#include "serialization/serialize/serializetype.cpp"

```

Comment:
Please do not include an cpp file.
You should separate it into the header files and the code files.

Reply:
added the functions in loader.h file .. Thanks